pkg/install/service.go

```diff
@@ -98,7 +98,13 @@ func EnsureService(args []string) error {
 
 	svcConfig.Dependencies = deps
 	svcConfig.Arguments = args
-
+	if force {
+		logrus.Infof("Uninstalling %s service", svcConfig.Name)
+		err = s.Uninstall()
+		if err != nil && err != service.ErrNotInstalled {
+			return fmt.Errorf("failed to uninstall service: %v", err)
```

Comment:
Maybe in the spirit of `--force`, this could be a logrus.Warnf and it would let it try the `s.Install()` 🤔